canonical-webteam/demoservice
# Initial support for docker-compose

## DONE
Add initial support for docker-compose in order to make it easier to test changes locally.
This will also help us get a closer environment in development to the one we have live.

## QA
- Run `docker-compose up` and verify you can access the interface on localhost:8099.
